### ISSUE TITLE
mix dogma crashed 

### ISSUE THREAD

**Issue by bansalakhil** [lpil/dogma#181]
running `mix dogma` in an umbrella app crashed. 

```
akhil@akhil-thinkpad:~/elixir/vchat_umbrella$ mix dogma
Inspecting 105 files.

** (EXIT from #PID<0.48.0>) an exception was raised:
    ** (FunctionClauseError) no function clause matching in Dogma.Rule.QuotesInString.check_binary_strings/2
        lib/dogma/rule/quotes_in_string.ex:25: Dogma.Rule.QuotesInString.check_binary_strings(nil, [])
        (elixir) lib/enum.ex:1088: Enum."-map/2-lists^map/1-0-"/2
        (elixir) lib/enum.ex:1088: Enum."-map/2-lists^map/1-0-"/2
        lib/dogma/script.ex:97: Dogma.Script.run_tests/2
        lib/dogma/rules.ex:22: Dogma.Rules.test_script/3
        (elixir) lib/task/supervised.ex:89: Task.Supervised.do_apply/2
        (elixir) lib/task/supervised.ex:40: Task.Supervised.reply/5
        (stdlib) proc_lib.erl:240: :proc_lib.init_p_do_apply/3

```

**Comment by lpil** [comment]
Hey Akhil. Can you provide any more information? Perhaps share the project?

**Comment by bansalakhil** [comment]
https://github.com/bansalakhil/vchat-umbrella

**Comment by lpil** [comment]
Thank you.

**Comment by lpil** [comment]
Hey Akhil, is this definitely the right codebase? Dogma runs fine, both the latest version on Hex, and the version on the master branch.

**Comment by lnikkila** [comment]
I got this as well, I had a syntax error in my code. Seems like any syntax error triggers the same exception.

**Comment by lpil** [comment]
Ah, I see. Thanks for the report. :)

**Comment by lpil** [comment]
Fixed #176, will wait for @bansalakhil to confirm that this is the same problem.

**Comment by lnikkila** [comment]
Good stuff, thanks for your time!
